test/NuGet.Core.Tests/NuGet.Build.Tasks.Console.Test/MSBuildStaticGraphRestoreTests.cs

```diff
@@ -126,7 +127,33 @@ public void GetPackageDownloads_WhenNotExactVersion_ThrowsException(string versi
                     var _ = MSBuildStaticGraphRestore.GetPackageDownloads(project).ToList();
                 };
 
-                act.Should().Throw<ArgumentException>().WithMessage($"'{expected ?? VersionRange.Parse(version).OriginalString}' is not an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.");
+                act.Should().Throw<ArgumentException>().WithMessage($"Package 'PackageA {expected ?? version}' does not have an exact version like '[1.0.0]'. Only exact versions are allowed with PackageDownload.");
```

Comment:
Based on feedback elsewhere, I think this is more robust:

```C#
string expectedMessage = string.Format(Strings.Error_PackageDownload_OnlyExactVersionsAreAllowed, expected ?? version, "PackageA");

act.Should().Throw<ArgumentException>().WithMessage(expectedMessage);
```

This way, the test's expected message isn't hard-coded to en-US strings.

You did this already in new tests, like [here](https://github.com/NuGet/NuGet.Client/pull/5351/files#diff-071c19f558be6f4bbcd2a336dcde0c31f3f56d38f013a92fed62e6f5206bd598R156).

Reply:
Thank you! I have removed the hard coded en-US strings.